src/psyclone/dynamo0p3.py

```diff
@@ -7153,11 +7153,11 @@ def gen_stub(self):
 
         '''
         # Check iteration space before generating code
-        if self._iterates_over not in USER_KERNEL_ITERATION_SPACES:
+        if self.iterates_over not in USER_KERNEL_ITERATION_SPACES:
             raise InternalError(
                 "DynKern.gen_stub(): Expected one of {0} as iteration "
                 "space but found '{1}' in kernel '{2}'.".
-                format(USER_KERNEL_ITERATION_SPACES, self._iterates_over,
+                format(USER_KERNEL_ITERATION_SPACES, self.iterates_over,
```

Comment:
Sorry to be a pain but we might as well do the `self._name` on the line below too.

Reply:
Done.